crates/sui-bcs-json/src/lib.rs

```diff
@@ -281,11 +350,7 @@ fn internal_bcs_from_json(typename: &str, data: &str) -> Result<Vec<u8>> {
 
 fn to_bcs_from_vector(typename: &str, data: &str) -> Result<Vec<u8>> {
     let mut output = vec![];
-    let typename: &str = typename
-        .split(&['<', '>'][..])
-        .collect::<Vec<_>>()
-        .get(1)
-        .unwrap();
+    let typename = is_vector(typename).unwrap();
```

Comment:
this is safe to unwrap because typename is first checked if it is a vector before being passed to this function.